dates/extended/perfect_A.json

```diff
@@ -0,0 +1,67 @@
+[
+    [
```

Comment:
I'd suggest nesting this in an object instead of a list. And then name the list of mons 'list' or something. And then store the list of different test scenarios in a list.